docs/src/components/IconsList.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import {
+  Button,
+  Card,
+  CardSection,
+  Coupon,
+  Heading,
+  InputField,
+  Stack,
+  Text,
+  Tile,
+} from "@kiwicom/orbit-components";
+import { Link as LinkIcon, Search } from "@kiwicom/orbit-components/icons";
+import iconsList from "@kiwicom/orbit-components/lib/data/icons.json";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+import { CodeBlock } from "./Code";
+import { copyTimeout, pascalize } from "../utils/common";
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
+  grid-gap: ${({ theme }) => theme.orbit.spaceXSmall};
+`;
+
+const StyledAnchorWrapper = styled.a`
+  a {
+    float: left;
+    padding-top: 4px;
+    padding-right: 2px;
+    margin-left: -22px;
```

Comment:
i tried it without that negative margin and it looks better. Why you need that? 🤔

Reply:
Without the negative margin, the header is pushed to the right and doesn't align with the elements below it. Because the anchor link will mostly be hidden, I thought that was more important.

Reply:
Moved to a ButtonLink on the right of the head, so this is all unnecessary.